src/Kiota.Builder/CodeDOM/CodeTerminalWithKind.cs

```diff
@@ -4,7 +4,9 @@
 namespace Kiota.Builder.CodeDOM;
 
 public abstract class CodeTerminalWithKind<T> : CodeTerminal where T : Enum {
+#nullable disable
     public T Kind { get; set; }
+#nullable enable
```

Comment:
Is this directive localized? i.e. Would it be possible to write `item.Kind = null`?

Reply:
This is a limitation of the generic types constraint, if you look closely, it can only be an enum, which means it can't be null by definition.